releasetasks/templates/l10n.yml.tmpl

```diff
@@ -0,0 +1,32 @@
+{% for platform in l10n_platforms %}
+# TODO: iterate over platform specific locales...probably need l10n json to do this
+{% for locale, locale_revision in l10n_changesets.iteritems() %}
+# TODO: make a helper function to generate consistent builder names?
+{% set buildername = "{}_{}_{}_l10n_repack".format(branch, product, platform) %}
+-
+    taskId: "{{ stable_slug_id(buildername) }}"
```

Comment:
The original version had chunked builders, but Rail and I realized that it would mean that both this code and the script running the job would have to chunk the locales on their own. I didn't want to take the risk of those getting out of sync and triggering downstream jobs for the wrong locales when one chunk completed.

I don't think we need unique taskIds for each locale, but having multiple locales in one task means that we can't trigger downstream funsize/beetmover/etc. on them until the entire task completes successfully. When they do, we can branch out each repack task into multiple sets of funsize, beetmover, et. al.

At one point we talked about doing one task per locale, but Catlee thought that would end up being very inefficient. Once we're using the new system it should be pretty easy to tweak the the chunks and find out though :)